src/app/fdctl/config.h

```diff
@@ -269,7 +269,7 @@ typedef struct {
       char  snapshot[ PATH_MAX ];
       char  status_cache[ PATH_MAX ];
       ulong tpool_thread_count;
-      uint  cluster_version;
+      char  cluster_version[ PATH_MAX ];
```

Comment:
PATH_MAX is overkill, you can use 32 or so here. The TOML parser automatically picks up the max string length.